### PR TITLE
343 Correcting simulations' start scripts

### PR DESCRIPTION
**Pull request by mknaranja**
## Merge Request - GuideLine Checklist 

**Guideline** to check code before resolve WIP and approval, respectively.
As many checkboxes as possible should be ticked.

### Checks by code author:
* [x] There is at least one issue associated with the pull request.
* [x] The branch follows the naming conventions as defined in the [git workflow](git-workflow).
* [x] New code adheres with the [coding guidelines](coding-guidelines)
* [ ] Tests for new functionality has been added
* [ ] A local test was succesful
* [x] There is appropriate **documentation** of your work. (use doxygen style comments)
* [ ] If new third party software is used, did you pay attention to its license? Please remember to add it to the wiki after successful merging.
* [ ] If new mathematical methods or epidemiological terms are used, has the glossary been updated ? Did you provide further documentation ?
 is present or referenced. Please provide your references.
* [ ] The following questions are addressed in the documentation*:  Developers (what did you do?, how can it be maintained?), For users (how to use your work?), For admins (how to install and configure your work?)
* For documentation: Please write or update the Readme in the current working directory!

### Checks by code reviewer(s):
* [x] Is the code clean of development artifacts e.g., unnecessary comments, prints, ...
* [x] The ticket goals for each associated issue are reached or problems are clearly addressed (i.e., a new issue was introduced).
* [ ] There are appropriate **unit tests** and they pass.
* [ ] The git history is clean and linearized for the merge request.
* [x] Coverage report for new code is acceptable.

### CODE REVIEW

**Comment by mknaranja** [comment]
Small changes and some corrected auto formatting

**Comment by mknaranja** [comment]
@HenrZu @dabele Please see independent commits for changed formatting

File: cpp/simulations/paper_202107.cpp [hunk 1]
```diff
@@ -583,16 +587,15 @@ mio::IOResult<void> set_edges(const fs::path& data_dir,
...
+                      mio::read_mobility_plain((data_dir / "mobility" / "commuter_migration_scaled.txt").string()));
     BOOST_OUTCOME_TRY(migration_data_twitter,
-                      mio::read_mobility_plain((data_dir / "migration" / "twitter_scaled_1252.txt").string()));
+                      mio::read_mobility_plain((data_dir / "mobility" / "twitter_scaled_1252.txt").string()));
     if (migration_data_commuter.rows() != Eigen::Index(params_graph.nodes().size()) ||
         migration_data_commuter.cols() != Eigen::Index(params_graph.nodes().size()) ||
         migration_data_twitter.rows() != Eigen::Index(params_graph.nodes().size()) ||
         migration_data_twitter.cols() != Eigen::Index(params_graph.nodes().size())) {
-        return mio::failure(mio::StatusCode::InvalidValue, "Migration matrices not the correct size.");
+        return mio::failure(mio::StatusCode::InvalidValue, "Mobility matrices do not have the correct size.");
```
Comment (HenrZu) [inline cpp/simulations/paper_202107.cpp:598]: Maybe its helpful at this point to refer to the existing file` transformMobilityData.py` which merges the rows and columns of Eisenach to Wartburgkreis (dim 401 x 401 -> 400x400).

**Review by HenrZu — COMMENTED** [review]
Looks good. Added 2 comments.

File: cpp/simulations/paper_202107.cpp (hunk 1 above)
Comment (HenrZu) [inline cpp/simulations/paper_202107.cpp:598]:
```suggestion
        return mio::failure(mio::StatusCode::InvalidValue,
                            "Mobility matrices do not have the correct size. You may need to run "
                            "transformMobilitydata.py from pycode memilio epidata package.");
```

**Review by HenrZu — APPROVED** [review]

**Pull request merged**